### PR TITLE
Added testimonials

### PR DESCRIPTION
**Pull request by Lavanyaaa25**
Added testimonials
 Issue#20

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,6 +160,61 @@ <h2 class="media-heading  text-center" style="color:black;">Mentorship</h2>
...
+    <div class="row">
+      <div class="col-md-12">
+        <h2 class="section-title wow fadeInUp" data-wow-delay="0.3s">Testimonials</h2>
+      </div>
+      <div class="col-md-4">
+        <div class="testimonial-box wow fadeInUp" data-wow-delay="0.4s">
+          <div class="testimonial-author">
+            <img src="images/testi_1.jpg" class="img-circle" alt="Testimonial Author">
+            <h4>John Doe</h4>
+            <p>CEO, ABC Company</p>
```
Comment (Its-Aman-Yadav) [inline index.html:176]: Could you please remove that ABC or XYZ or something company name,
it will prefer to not to have a company name either

**Review by Its-Aman-Yadav — CHANGES_REQUESTED** [review]

**Comment by Lavanyaaa25** [comment]
Sure , will remove that.

**Comment by Lavanyaaa25** [comment]
Screenshot 2024-05-12 183308
Done

File: index.html (hunk 1 above)
Comment (Lavanyaaa25) [reply index.html]: I have removed it.Please accept

**Comment by Its-Aman-Yadav** [comment]
Now its looking good, thank you so much @Lavanyaaa25 for your contribution :)

**Comment by Lavanyaaa25** [comment]
Can u add the labels gssoc and level for it to. Count on leaderboard

**Pull request merged**